generators/app/templates/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module '*.png';
+declare module '*.jpg';
+declare module '*.json';
+declare module 'reactotron-apisauce';
+declare module 'recompose';
+declare module 'redux-recompose';
+declare module 'redux-persist-seamless-immutable';
```

Comment:
this is optional, it comes with `redux-persist` feature

Reply:
Done